spinoso-time/src/time/tzrs/mod.rs

```diff
@@ -5,15 +5,20 @@ use tz::datetime::DateTime;
 
 mod build;
 mod convert;
+mod error;
 mod math;
 mod offset;
 mod parts;
 mod timezone;
 mod to_a;
 
+pub use error::TimeErr;
 pub use offset::Offset;
 pub use to_a::ToA;
 
+/// Alias for [`std::result::Result`] with the unified `TimeErr`
+pub type Result<T> = std::result::Result<T, TimeErr>;
```

Comment:
Idiomatic Rust is to call this `TimeError` instead of `TimeErr`. The `Err` associated type in `FromStr` is considered a mistake whereas the `Error` associated type in `TryFrom` is not.